src/Entity/JoindInUser.php

```diff
@@ -35,28 +35,25 @@ class JoindInUser implements \JsonSerializable
      * @var string
      */
     private $displayName;
+    /**
+     * @ORM\Column(type="boolean", nullable=false, options={"default":false})
+     *
+     * @var bool
+     */
+    private $organizer;
 
     /**
      * @var \Doctrine\Common\Collections\ArrayCollection
      * @ORM\OneToMany(targetEntity="App\Entity\JoindInComment", mappedBy="user")
      */
     private $comments;
 
-    public function __construct(int $id, string $username, string $displayName)
+    public function __construct(int $id, string $username, string $displayName, bool $isOrganizer = false)
```

Comment:
I've deviated from the original property name trying to make the meaning of the property more clear when using IDE argument hints.